### DISCUSSION TITLE
Scramble Mod

### DISCUSSION THREAD

**Discussion by AnxietyWump** [ppy/osu#13126] [Ideas]
**Describe the new feature:** 
Randomly came up with this **outrageous** idea for a mod I call “Scramble”. The mod will misplace a circle or slider occasionally by a *reasonable* distance to make sure it’s not insane. While also ensuring the timing is still correct (as if the mod wasn’t active).

**Proposal designs of the feature:** 
The icon/sprite for the mod could be a clutter or mess of circles and sliders scattered.

Thank you!

**Comment by enoslayd** [comment]
Something like random mod on mania but for standard instead?

**Comment by AnxietyWump** [comment]
> Something like random mod on mania but for standard instead?

No idea what that is because I don't play mania, but most likely yeah,

**Comment by enoslayd** [comment]
It randomizes note placements
